community/contribution_guide/release.md

```diff
@@ -0,0 +1,631 @@
+```bash
+# Check the version, which should be 2. x
+gpg --version 
+```
+
+### Install SHASUM
+Used to generate signatures for files.
+
+### SVN
+Used to pull the Apache Release SVN repository. Apache uses svn to host the release of the project.
+
+### MAVEN
+Used to compile the project.
+
+## Environment Configuration
+### Configure GPG KEY
+#### Create the Key
+```shell
+gpg --gen-key
+```
+```bash
+gpg (GnuPG) 2.0.22; Copyright (C) 2013 Free Software Foundation, Inc.
+This is free software: you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+
+Please select what kind of key you want:
+   (1) RSA and RSA (default)
+   (2) DSA and Elgamal
+   (3) DSA (sign only)
+   (4) RSA (sign only)
+Your selection? 1
+RSA keys may be between 1024 and 4096 bits long.
+What keysize do you want? (3072) 4096 
+Requested keysize is 4096 bits
+Please specify how long the key should be valid.
+         0 = key does not expire
+      <n>  = key expires in n days
+      <n>w = key expires in n weeks
+      <n>m = key expires in n months
+      <n>y = key expires in n years
+Key is valid for? (0) 0 
+Key does not expire at all
+Is this correct? (y/N) y 
+```
+
+As shown above, the options are:
+- 1
+- 4096
+- 0
+- y
+
+In the process of configuring GPG, the key types selected are RSA and RSA (default), so the first option is 1, and the desired key size can be directly input the corresponding value. The third option is 0, indicating that the validity period of the key is never expired. Finally, input y to ensure that the above input is correct.
+
+```shell
+You need a user ID to identify your key; the software constructs the user ID
+from the Real Name, Comment and Email Address in this form:
+    "Heinrich Heine (Der Dichter) <heinrichh@duesseldorf.de>"
+
+Real name: tyrantlucifer 
+Email address: tyrantlucifer@apache.org
+Comment: The key of Apache SeaTunnel
+You selected this USER-ID:
+    “tyrantlucifer (The key of Apache SeaTunnel) <tyrantlucifer@apache.org>”
+
+Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? O 
+You need a password to protect your private key.
+```
+
+As shown above, you will need to specify your personal information and encryption password for this key. You will need to fill in the following information:
+- Name
+- Email (Apache Email)
+- Comment for key
+- Password (Very important, don't forget)
+
+
+```shell
+We need to generate a lot of random bytes. It is a good idea to perform
+some other action (type on the keyboard, move the mouse, utilize the
+disks) during the prime generation; this gives the random number
+generator a better chance to gain enough entropy.
+
+gpg: key 0983DF85 marked as ultimately trusted
+public and secret key created and signed.
+
+pub   4096R/0983DF85 2022-12-28
+keys fingerprint = AE63 FC40 ECCD 600D 724B  5625 05FD AE73 0983 DF85
+uid                  tyrantlucifer (The key of Apache SeaTunnel) <tyrantlucifer@apache.org>
```

Comment:
I see you use `tyrantlucifer` as an example, Can you add a explain about it in the begin of this document? Let user know `tyrantlucifer` only is an example and the release manager need use they own userid. On the other hand, you need hide the key about `tyrantlucifer` ensure don't leakage of critical information.

Reply:
I have done it, PTAL!